src/main/java/me/tsukanov/counter/ui/CounterFragment.java

```diff
@@ -212,6 +226,15 @@ public void increment() {
         }
     }
 
+    public void incrementDecade() {
+        if (value < MAX_VALUE) {
+            setValue(value+10);
+            vibrate(DEFAULT_VIBRATION_DURATION + 10);
```

Comment:
This value should be a constant.

Reply:
Yes it could be but I don't think it's less clear this way and we don't use it much.